src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/CaseOfficerPersonalisationFactory.java

```diff
@@ -36,4 +39,36 @@ public CaseOfficerPersonalisationFactory(
                 .put("Hyperlink to user’s case list", iaCcdFrontendUrl)
                 .build();
     }
+
+    public Map<String, String> createListedCase(
+        AsylumCase asylumCase
+    ) {
+        requireNonNull(asylumCase, "asylumCase must not be null");
+
+        hearingDateTime = asylumCase.read(AsylumCaseDefinition.HEARING_DATE_TIME, String.class).orElse("");
+
+        final String hearingDate =
+            LocalDate
+                .parse(extractHearingDate(hearingDateTime))
+                .format(DateTimeFormatter.ofPattern("d MMM yyyy"));
+
+        return
+            ImmutableMap
+                .<String, String>builder()
+                .put("Appeal Ref Number", asylumCase.read(AsylumCaseDefinition.APPEAL_REFERENCE_NUMBER, String.class).orElse(""))
+                .put("Listing Ref Number", asylumCase.read(AsylumCaseDefinition.ARIA_LISTING_REFERENCE, String.class).orElse(""))
+                .put("Hearing Date", hearingDate)
+                .put("Hearing Time", extractHearingTime(hearingDateTime))
+                .put("Hearing Centre Address", asylumCase.read(AsylumCaseDefinition.HEARING_CENTRE_ADDRESS, String.class).orElse(""))
+                .build();
+    }
+
+    private String extractHearingDate(String hearingDateTime) {
+        return hearingDateTime.substring(0, 10);
```

Comment:
Is this a guaranteed way of getting the correct date string?  I am assuming there is a standard date format being used so perhaps some validation should be performed to check we have the right string?  Otherwise it could be an invalid state.

Reply:
Resolved by parsing LocaDate and LocalTime and throwing DateTimeParseException.